src/apis/logs_api.ts

```diff
@@ -43,4 +134,56 @@ export class LogsAPI {
 
         return parser.parseDecodedLogs(decodedLogs);
     }
+
+    /**
+     * TODO: Add watch function.
```

Comment:
won't this get pulled into our documentation? is that intended?

Reply:
Nope, since it's not an actual function. I think having TODOs are fine.

Reply:
cool